scripts/rc.py

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/python3
-import os, sys, argparse, requests, base64, subprocess
+import  argparse, base64,  os, requests, subprocess, sys
+
+import cloudstorage as gcs
```

Comment:
Do we need these in this PR if the cloud stuff is coming in a later PR

Reply:
ah you're right, I missed those